src/glue.d

```diff
@@ -242,93 +262,97 @@ void obj_start(char *srcfile)
     rtlsym_reset();
     clearStringTab();
 
-#if TARGET_WINDOS
-    // Produce Ms COFF files for 64 bit code, OMF for 32 bit code
-    assert(objbuf.size() == 0);
-    objmod = global.params.mscoff ? MsCoffObj::init(&objbuf, srcfile, NULL)
-                                  :       Obj::init(&objbuf, srcfile, NULL);
-#else
-    objmod = Obj::init(&objbuf, srcfile, NULL);
-#endif
+    version (Windows)
```

Comment:
Right, but the trouble here is the object file support for other targets is build-specific.